pkg/backup/volume_snapshot_action_test.go

```diff
@@ -155,6 +155,30 @@ func TestVolumeSnapshotAction(t *testing.T) {
 			pv:              `{"apiVersion": "v1", "kind": "PersistentVolume", "metadata": {"name": "mypv"}, "spec": {"gcePersistentDisk": {"pdName": "pd-abc123"}}}`,
 			expectError:     true,
 		},
+		{
```

Comment:
Why do we need these 2 additional test cases - how are they different from what's above?

Reply:
These are not different from the other test cases and are repetitive tests. Will remove them.